pages/index.tsx

```diff
@@ -7,48 +7,57 @@ import { use, useEffect, useMemo } from 'react'
 import { GoogleMap, useLoadScript, Marker, useJsApiLoader } from '@react-google-maps/api'
 import { env } from 'process'
 import { useState } from 'react';
-import FindLocation from '@/hooks/FindLocation' 
-
+import FindLocation from '@/hooks/FindLocation'
+import Map from '@/components/Map'
 
 export default function Home() {
+
   //These are the two states used to get our location for centering
-  const [location, setLocation] = useState()
+  const [location, setLocation] = useState(null)
   const [hasLoaded, setHasLoaded] = useState(false)
+  const [showInfo, setShowInfo] = useState(false)
 
-// this sets our location State using this function
+  // this sets our location State using this function
   function setUserLocation() {
     return FindLocation().then((value) => {
       setLocation(value)
       return value
     })
   }
 
-//This is a useEffect used to make sure that the users location is grabbed only once when the page is rendered
+  //This is a useEffect used to make sure that the users location is grabbed only once when the page is rendered
   useEffect(() => {
+    fetch('api/pointsOfInterests').then((data) => {
+      console.log(data)
+    })
     if (!hasLoaded) {
-      setUserLocation()
+      // setUserLocation()
       setHasLoaded(true)
     }
   }, [])
-  
-  
+
+
   // this is our key and how we load in our google maps api
   const key: any = process.env.NEXT_PUBLIC_GOOGLE_MAPS_API_KEY
   const { isLoaded } = useLoadScript({
     'googleMapsApiKey': key
   })
-  
+
   //sets the space where the map should be to loading... if it is not yet rendered
-  if(!isLoaded) return <div>Loading...</div>
-  
+  if (!isLoaded) return <div>Loading...</div>
+
   console.log(location)
 
-  //This is where we make the map. It has to be inside the Home functional component so that it has access to the location state used to set the intitial start.
-  function Map(){
-    const center = useMemo(() => (location), [])
-    return <GoogleMap options={{disableDefaultUI: true,}} zoom={10} center={center} mapContainerClassName='map-container'></GoogleMap>
+  //start of api maybe
```

Comment:
lol
[sand] Make this comment closer to the api call and have it actually describe what the api does

Reply:
fixed